packages/components/src/ActionButton.tsx

```diff
@@ -2,18 +2,31 @@ import clsx from "clsx";
 import { createElement } from "react";
 import { tv, type VariantProps } from "tailwind-variants";
 
+const colors = {
+  primary: "yellow",
+  secondary: "cyan",
+  black: "black",
+} as const;
+
+type HoverColor = {
+  hoverColor?: keyof typeof colors;
+};
+
 const actionButton = tv({
   base: clsx(
     `group flex items-center cursor-pointer min-h-[2em] overflow-hidden text-black text-center relative`,
-    `transition-colors duration-100 w-full select-none justify-center font-medium`,
+    `[transition:color_0.1s,background_0.5s] ease-out-circ w-full select-none justify-center font-medium`,
```

Comment:
Unfortunately, `[transition:color_0.1s,background_0.5s]` overrides the easing :( Maybe we could just force the `ease-out-circ` prefixing it with a `!`. Do you know a better solution rather than write the whole cubic-bezier function here?

Reply:
I created hover-fill class in base.css to apply easing